### ISSUE TITLE
Allow the code to specify/check if Celest is running locally or in the cloud (and where in the cloud).

### ISSUE THREAD

**Issue by marcglasberg** [celest-dev/celest#43]
When you `init` the app:

```dart
celest.init(env: CelestEnvironment.local);
celest.init(env: CelestEnvironment.staging);
celest.init(env: CelestEnvironment.prod);
```

Where `CelestEnvironment` is autogenerated with the client as an `enum`.

Then, I can access it from BOTH the frontend and the backend code, like this: `celest.env` 

This has a few uses:

* I can show this info in the app UI, just like the "debug banner" Flutter shows. 
* I can simulate third-party services, when I'm running `local`.
* I can use staging versions of third-party services, when I'm running `staging`.
* I can run the real third-party services, when I'm running `prod`.

```dart
enum CelestEnvironment {
  local, staging, prod;

  bool get isLocal => this == CelestEnvironment.local;
  bool get isCloud => this == CelestEnvironment.staging || this == CelestEnvironment.prod;
}
```

**👉 Extra points if I can later run `celest.init()` again to plug another environment while the app is running.**

**Comment by dnys1** [comment]
Love this! We're going to prioritize this for the `0.2.0` release.

**Comment by dnys1** [comment]
This has been fixed in 0.2.0 🚀
